src/NServiceBus.Persistence.TestRunner/NServiceBus.Persistence.TestRunner.csproj

```diff
@@ -1,107 +1,27 @@
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Fabric, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
-      <HintPath>..\packages\Microsoft.ServiceFabric.5.5.216\lib\net45\System.Fabric.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Fabric.Management.ServiceModel, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
-      <HintPath>..\packages\Microsoft.ServiceFabric.5.5.216\lib\net45\System.Fabric.Management.ServiceModel.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Fabric.Management.ServiceModel.XmlSerializers, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.ServiceFabric.5.5.216\lib\net45\System.Fabric.Management.ServiceModel.XmlSerializers.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Fabric.Strings, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
-      <HintPath>..\packages\Microsoft.ServiceFabric.5.5.216\lib\net45\System.Fabric.Strings.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Runtime.Serialization" />
-    <Reference Include="System.ServiceModel" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="CommunicationListener.cs" />
-    <Compile Include="CompositeCommunicationListener.cs" />
-    <Compile Include="CompositeListener.cs" />
-    <Compile Include="EventSourceLogger.cs" />
-    <Compile Include="EventSourceTestListener.cs" />
-    <Compile Include="INeed.cs" />
-    <Compile Include="ITestRunner.cs" />
-    <Compile Include="ResultListener.cs" />
-    <Compile Include="NeedAttribute.cs" />
-    <Compile Include="Result.cs" />
-    <Compile Include="StatefulServiceProviderListener.cs" />
-    <Compile Include="AbstractTestRunner.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+    <PackageReference Include="Microsoft.ServiceFabric" Version="5.6.210" />
+    <PackageReference Include="Microsoft.ServiceFabric.Data" Version="2.6.210"/>
+    <PackageReference Include="Microsoft.ServiceFabric.FabricTransport.Internal" Version="2.6.210"/>
+    <PackageReference Include="Microsoft.ServiceFabric.Services" Version="2.6.210"/>    
+    <PackageReference Include="Microsoft.ServiceFabric.Services.Remoting" Version="2.6.210"/>        
+    <PackageReference Include="NUnit" Version="3.7.1"/>            
+    <PackageReference Include="NUnitLite" Version="3.7.1"/>                
```

Comment:
Test runner details because we are running tests inside the cluster. This functionality is only available in NUnit Lite